public/css/main.scss

```diff
@@ -32,31 +31,31 @@ body {
 // Body left and right margin
 @media screen and (min-width: 1408px) {
   body {
-    margin: 0 25rem;
+    margin: 0 25rem !important;
```

Comment:
Don't use important! Just move the custom CSS to the end of the file (after the bulma imports).